test/rekt/channel_test.go

```diff
@@ -145,3 +145,36 @@ func TestSmoke_ChannelImplWithSubscription(t *testing.T) {
 		)
 	}
 }
+
+/*
+TestChannelChain tests the following scenario:
+
+EventSource ---> (Channel ---> Subscription) x 10 ---> Sink
+
+*/
+func TestChannelChain(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment(
+		knative.WithKnativeNamespace(system.Namespace()),
+		knative.WithLoggingConfig,
+		knative.WithTracingConfig,
+		k8s.WithEventListener,
+		environment.Managed(t),
+	)
+
+	env.Test(ctx, t, channel.ChannelChain(10))
+}
+
+func TestChannelDeadLetterSink(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment(
+		knative.WithKnativeNamespace(system.Namespace()),
+		knative.WithLoggingConfig,
+		knative.WithTracingConfig,
+		k8s.WithEventListener,
```

Comment:
need a `environment.Managed(t),` here

```suggestion
		k8s.WithEventListener,
		environment.Managed(t),
```